web/locales/translation-status.js

```diff
@@ -0,0 +1,65 @@
+const { reverse, flatMap, difference, size, sortBy } = require('lodash');
```

Comment:
I would recommend moving this file to web/translation-status.js as web/locales should only contain the translation files themselves.
We would also need to update the Dockerfile to make sure that this file is indeed added to the build.